README.md

```diff
@@ -35,6 +35,12 @@ the review process quicker and more streamlined.
 
 ## Install
 
+To use The Lounge you must have [Node.js](https://nodejs.org/en/download/) installed.
+The oldest Node.js version we support is 4.2.0.
```

Comment:
That's not what I meant. The major releases, not any release in April.

I think you misunderstand how LTS works. The point of LTS is that the release branch will be updated with bug fixes and security updates.

4.4.3 isn't LTS. That doesn't mean anything. 4.4.3 is, and always will be, exactly the same code. It will never change. So how can there be any support for it?

What is LTS is the 4.x release branch. They word it weirdly in node, but the point is that 4.x will get bug fixes and security updates until it is no longer LTS (next April).

When a major April release happens (4.x was different in that it was released in September, but from now on all the LTS releases will be from April major releases), 6 months later it becomes LTS, and is the stablest of the stable that they have, and will be updated with bug fixes and security updates for 18 months.

So, it doesn't mean anything to say "4.2.0 is LTS" or "4.4.3 is LTS". No, 4.x is LTS. That's all.